core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/PostProcessingSimulation.kt

```diff
@@ -98,18 +97,18 @@ fun buildFinalEnvelope(
                     graph,
                     newEnvelope,
                     blockAvailability,
-                    ranges,
+                    edges,
                     departureTime,
                     stops
                 ) ?: return newEnvelope
             if (fixedPoints.any { it.offset == conflictOffset })
                 break // Error case, we exit and fallback to the linear envelope
             logger.info(
-                "Conflict in new envelope at offset {}, splitting mareco ranges",
+                "Conflict in new envelope at offset {}, splitting mareco edges",
```

Comment:
This does refer to ranges, just not the same kind.

But the message is a little outdated anyway, we could change it that way instead
(this may break the file formatting, I'm editing from the github UI)

```suggestion
                "Conflict when running final stdcm simulation at offset {}, adding a fixed time point",
```